tools/qfvis/source/flow.c

```diff
@@ -336,6 +564,18 @@ RecursiveClusterFlow (int clusternum, threaddata_t *thread, pstack_t *prevstack)
 		thread->base->numcansee++;
 	}
 
+	// check all target portals instead of just neighbor portals, if the time is right
```

Comment:
"if the time is right" doesn't have much meaning here. What are the conditions for the time being right? Why?